modules/text/include/opencv2/text/erfilter.hpp

```diff
@@ -227,6 +227,11 @@ features: hole area ratio, convex hull ratio, and number of outer inflexion poin
 CV_EXPORTS_W Ptr<ERFilter> createERFilterNM2(const Ptr<ERFilter::Callback>& cb,
                                                   float minProbability = (float)0.3);
 
+CV_EXPORTS_W  Ptr<ERFilter> createERFilterNM1(const String& filename,
```

Comment:
Documentation in our case is comments of spacial type. Take a look at the previous constructor in this file. I'ts [documented](https://github.com/abratchik/opencv_contrib/blob/59df3bf5421c95dda7ddd22cbf6c78f23c647e41/modules/text/include/opencv2/text/erfilter.hpp#L216-L226).